src/App.tsx

```diff
@@ -1,26 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const items: string[] = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [amountOfItems, setAmountOfItems] = useState(5);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [info, setInfo] = useState(`Page 1 (items 1 - 5 of ${items.length})`);
+
+  const startItem: number = currentPage * amountOfItems;
+  const pages: number[] = [];
+
+  const newList: string[] = items.slice(startItem, startItem + amountOfItems);
+
+  for (let i = 1; i <= Math.ceil(items.length / amountOfItems); i += 1) {
```

Comment:
Avoid using a `for` loop to create a new array with each element increased by one. Instead, use `map`.